sdk/cosmosdb/cosmos/src/diagnostics/CosmosDiagnostics.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getCosmosDiagnosticsToString, getdiagnosticsdurationMilliseconds } from "./Diagnostics"; //static?
+export class CosmosDiagnostic {
```

Comment:
Filename and class neme are inconsistent - my preference would be using CosmosDiagnostic**s** for consistency sake in the class name as well

Reply:
Done